### PR TITLE
Update showHud setter for Squad Radar

### PR DESCRIPTION
**Pull request by jonpas**
**When merged this pull request will:**
- Replaces #4043 for `master` branch (didn't make it into 1.62)

### CODE REVIEW

**Comment by Glowbal** [comment]
Can this go in for 3.7.0?

**Comment by jonpas** [comment]
No, it's not in 1.62, as they pulled it out just before release. No idea if it will be in 1.64 either.

**Comment by commy2** [comment]
3.8.0 then?

**Comment by jonpas** [comment]
Aye.

**Comment by commy2** [comment]
krgr

**Comment by jonpas** [comment]
If anyone currently on RC branch can check that up to verify it's there, that would be great.

**Comment by jonpas** [comment]
This is good to go, need review.

**Comment by PabstMirror** [comment]
Did it make 1.64? didn't see it in the patch notes

**Comment by jonpas** [comment]
Yes, just checked, `shownHud` returns 9 elements.

File: addons/common/functions/fnc_showHud.sqf [hunk 1]
```diff
@@ -29,7 +30,12 @@
 
 if (!hasInterface) exitWith {[-1]};
 
-params [["_reason", "", [""]], ["_mask", [], [[]], [0,8]]];
+params [["_reason", "", [""]], ["_mask", [], [[]], [0,8,9]]];
+
+if ((count _mask) == 8) then {
+    ACE_LOGWARNING_1("ace_common_fnc_showHud - mask now takes 9 arguments in 1.62 [called with %1]",_this);
```
Comment (commy2) [inline addons/common/functions/fnc_showHud.sqf]: #4282

**Comment by PabstMirror** [comment]
@jonpas 
The setting will have no effect as this isn't in 1.64
Should we make the setting non-client-settable until 1.66 comes out?

**Comment by jonpas** [comment]
Oh :S
I thought it is in 1.64, I'll revert this merge then, no one is using it with our setter anyways as it does nothing.

**Comment by jonpas** [comment]
Someone said commenting is supposed to allow reopening of a PR.

**Comment by jonpas** [comment]
Replaced by #4491

**Pull request merged**